src/services/multicall/multicall-service.ts

```diff
@@ -170,9 +170,15 @@ export class MulticallService implements IMulticallService {
 }
 
 const SUPPORTED_CHAINS: ChainId[] = [
-  1, 3, 4, 5, 10, 14, 16, 18, 19, 25, 30, 31, 40, 42, 56, 66, 69, 97, 100, 106, 108, 114, 122, 128, 137, 250, 288, 321, 420, 592, 1088, 1284,
-  1285, 1287, 2001, 4002, 8217, 8453, 9000, 9001, 42161, 42170, 42220, 42262, 43113, 43114, 44787, 59144, 71401, 71402, 80001, 84531, 421611,
-  421613, 11155111, 1313161554, 1666600000,
+  1, 3, 4, 5, 10, 14, 16, 18, 19, 25, 30, 31, 40, 42, 42, 44, 46, 50, 51, 56, 57, 61, 66, 69, 82, 83, 97, 100, 106, 108, 109, 114, 122, 128, 137,
```

Comment:
What are these? You added a lot of chains, where did you get these ids from?

Reply:
All from here: https://github.com/mds1/multicall/blob/main/deployments.json